client/settings/general-settings-section/index.js

```diff
@@ -3,148 +3,30 @@
  */
 import React, { useContext } from 'react';
 import styled from '@emotion/styled';
-import { Card, CheckboxControl } from '@wordpress/components';
+import { Card } from '@wordpress/components';
 
 /**
  * Internal dependencies
  */
-import PaymentMethodFeesPill from 'wcstripe/components/payment-method-fees-pill';
-import CardBody from '../card-body';
 import UpeOptInBanner from './upe-opt-in-banner';
-import UpeToggleContext from '../upe-toggle/context';
-import {
-	useEnabledPaymentMethods,
-	useGetAvailablePaymentMethods,
-} from './data-mock';
-import PaymentMethodDescription from './payment-method-description';
 import SectionHeading from './section-heading';
-import PaymentMethodsMap from '../../payment-methods-map';
-import PaymentMethodSetupHelp from './payment-method-setup-help';
+import SectionFooter from './section-footer';
+import PaymentMethodsList from './payment-methods-list';
+import UpeToggleContext from 'wcstripe/settings/upe-toggle/context';
 
 const StyledCard = styled( Card )`
 	margin-bottom: 12px;
 `;
 
-const PaymentMethodsList = styled.ul`
```

Comment:
All of this has been extracted to the `PaymentMethodsList` component in `./payment-methods-list`